src/components/list/List.js

```diff
@@ -0,0 +1,13 @@
+import { useState } from 'react';
+import database from '../../global';
+import Book from '../book/Book';
+import './list.css';
+
+export default function List() {
+  const [data, setData] = useState(database);
+  return (
+    <div className="container-list">
+      {data.map((item) => <Book key={item.id} category={item.category} title={item.title} author={item.author} percentage={item.percent} chapter={item.chapter} setData={setData} data={data} id={item.id} />)}
```

Comment:
- [ ] [OPTIONAL] In my opinion, you are passing in too many props to the `Book` component, and it makes it difficult to keep track of what is going on. You are also more likely to create bugs in your code as a result of mistakes or even typo.
I suggest you pass the only current `item` in your map iteration along with the `data` and `setData` to your `Book` as that is all you need. Then in the `Book` component, you can destructure the `item` to get what you need. I leave a demonstration below for you 👇 

In `List.js`
```js
<div className="container-list">
  {data.map((item) => <Book key={item.id} item={item} setData={setData} data={data} />)}
</div>
```

In `Book.js` destructure the props like this 👇 
```js
export default function Book({
  item: {
    percentage, category, title, author, chapter, id,
  }, setData, data,
}) {
}
```